integrations/atlassian/jira/webhooks.go

```diff
@@ -84,39 +82,17 @@ type webhookListResponse struct {
 // https://developer.atlassian.com/server/jira/platform/webhooks/
 func getWebhook(ctx context.Context, l *zap.Logger, baseURL, token string) (int, bool) {
 	// TODO(ENG-965): Support pagination.
-	req, err := http.NewRequestWithContext(ctx, http.MethodGet, baseURL+"/rest/api/3/webhook", nil)
+	u := baseURL + "/rest/api/3/webhook"
+	resp, err := common.HTTPGet(ctx, u, "Bearer "+token)
 	if err != nil {
-		l.Warn("Failed to construct HTTP request to list Jira webhooks", zap.Error(err))
-		return 0, false
-	}
-
-	req.Header.Set("Authorization", "Bearer "+token)
-
-	resp, err := httpClient.Do(req)
-	if err != nil {
-		l.Warn("Failed to list Jira webhooks", zap.Error(err))
-		return 0, false
-	}
-	defer resp.Body.Close()
-
-	body, err := io.ReadAll(resp.Body)
-	if err != nil {
-		l.Warn("Failed to read Jira webhooks list response", zap.Error(err))
-		return 0, false
-	}
-
-	if resp.StatusCode != http.StatusOK {
-		l.Warn("Unexpected response to Jira webhooks list request",
-			zap.Int("status", resp.StatusCode),
-			zap.ByteString("body", body),
-		)
+		l.Warn("Failed to request Jira webhook", zap.Error(err))
 		return 0, false
 	}
 
 	var list webhookListResponse
-	if err := json.Unmarshal(body, &list); err != nil {
+	if err := json.Unmarshal(resp, &list); err != nil {
```

Comment:
Limit size

Reply:
no need, the body that was read (by the new common function) is already size-limited